challenges/classes.js

```diff
@@ -4,4 +4,39 @@
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+
+class CuboidMaker{
+    constructor(cube){
+    this.length=cube.length;
+    this.width=cube.width;
+    this.height=cube.height
+    }
+    volume() {return this.length*this.width*this.height;};
+    surfaceArea(){return 2*((this.length*this.width)+(this.length*this.height)+(this.width*this.height));}
```

Comment:
You don't need the inner parenthesis.
```suggestion
    surfaceArea(){return 2*(this.length*this.width+this.length*this.height+this.width*this.height);}
```
will work